### PR TITLE
Introducing JSX

### PR DESCRIPTION
**Pull request by zrakhimov**
Hello,

Translated "Introducing JSX" page into Uzbek. Please review

@umidbekk Not sure who else is actively reviewing, but mentioning you here

Let me know if any issues

Cheers!

Zokir

Screenshot at Jun 01 16-13-13

### CODE REVIEW

**Comment by avocadowastaken** [comment]
@doniyor2109 help to review pls 🙏

File: content/docs/introducing-jsx.md [hunk 1]
```diff
@@ -175,10 +175,10 @@ const element = {
...
+Bu obyektlar "React elementlari" hisoblanadi. Tasavvur qilish uchun siz bularni ekranda ko'rmoqchi bo'lgan narsalarni ma'lumotlari deb faraz qilsangiz bo'ladi. React bu obyektlarni o'qiydi va DOM ni qurish va uni yangilab turish uchun ishlatadi.
 
-We will explore rendering React elements to the DOM in the [next section](/docs/rendering-elements.html).
+Biz React elementlarini DOM da chizishni [keyingi bo'limda](/docs/rendering-elements.html) yoritamiz.
 
->**Tip:**
+>**Maslahat:**
 >
->We recommend using the ["Babel" language definition](https://babeljs.io/docs/en/next/editors) for your editor of choice so that both ES6 and JSX code is properly highlighted.
+>Biz yoqtirgan redaktoringizda ES6 va JSX kod to'g'ri yoritilishi uchun [Babel](https://babeljs.io/docs/en/next/editors) bilan sozlashingizni maslahat beramiz.
```
Comment (doniyor2109) [inline content/docs/introducing-jsx.md]:
```suggestion
>Siz yoqtirgan redaktoringizda ES6 va JSX kod to'g'ri yoritilishi uchun [Babel](https://babeljs.io/docs/en/next/editors) bilan sozlashingizni maslahat beramiz.
```

File: content/docs/introducing-jsx.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Bu ajoyib yozish uslubi na string na HTML ga o'xshaydi.
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+Bu JSX deb ataladi, ya'ni JavaScript yozish uslubi kengaytmasi. Biz UI ni qanday ko'rinishga ega bo'lishini tasvirlash uchun shuni React bilan qo'llagan holda ishlatishni maslahat beramiz. JSX sizga boshqa qolip tillarini eslatishi mumkin, lekin u JavaScript ning to'liq kuchi bilan keladi. 
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+JSX React "elementlari" ni hosil qiladi. Biz ularni DOM ga chizishni [keyingi bo'limda](/docs/rendering-elements.html) ko'rib chiqamiz. Quyida, siz boshlashingiz uchun kerak bo'ladigan JSX ning asoslari bilan tanishib chiqishingiz mumkin
```
Comment (doniyor2109) [inline content/docs/introducing-jsx.md]:
```suggestion
JSX React "elementlari" ni hosil qiladi. Biz ularni DOM ga chizishni [keyingi bo'limda](/docs/rendering-elements.html) ko'rib chiqamiz. Quyida, siz boshlashingiz uchun kerak bo'ladigan JSX ning asoslari bilan tanishib chiqishingiz mumkin.
```

File: content/docs/introducing-jsx.md [hunk 3]
```diff
@@ -1,36 +1,36 @@
 ---
 id: introducing-jsx
-title: Introducing JSX
+title: JSX Kirish
```
Comment (doniyor2109) [inline content/docs/introducing-jsx.md]:
```suggestion
title: JSX ga kirish
```

**Review by doniyor2109 — CHANGES_REQUESTED** [review]

**Comment by avocadowastaken** [comment]
@zrakhimov thanks for the PR! please check comments from @doniyor2109 regarding the grammar (I can't tell whether they should be merged or not)

File: content/docs/introducing-jsx.md (hunk 1 above)
Comment (zrakhimov) [reply content/docs/introducing-jsx.md]: agree

File: content/docs/introducing-jsx.md (hunk 3 above)
Comment (zrakhimov) [reply content/docs/introducing-jsx.md]: Fair enough

**Comment by zrakhimov** [comment]
@umidbekk @doniyor2109  thanks guys! feel free to let me know if any issues. I've accepted the suggestion

**Review by avocadowastaken — APPROVED** [review]
thank you for the contribution 🎉

**Comment by zrakhimov** [comment]
@umidbekk I think you missed adding github username beside the topic/PR# on the issue page?

**Comment by avocadowastaken** [comment]
> @umidbekk I think you missed adding github username beside the topic/PR# on the issue page?

Thanks! fixed it

**Pull request merged**